cw/capture_sha3_cw310.yaml

```diff
@@ -20,7 +22,7 @@ capture:
   # w/o DOM
   #offset: 860
   # w/ DOM
-  offset: 2300
+  offset: 800
```

Comment:
I removed the offset. Good catch. There is some leakage right in the beginning even for the masked version.
I assume this could be the bus transfer, as 

> [Masking](https://docs.opentitan.org/hw/ip/kmac/doc/#masking)
The message FIFO does not generate the masked message data. Incoming message bitstream is not sensitive to the leakage. If the EnMasking parameter is set and [CFG_SHADOWED.msg_mask](https://docs.opentitan.org/hw/ip/kmac/doc/#Reg_cfg_shadowed) is enabled, the message is masked upon loading into the Keccak core using the internal entropy generator. The secret key, however, is stored as masked form always.

I will discuss that in https://github.com/lowRISC/opentitan/issues/15470 once everything is streamlined in the setup and scripts.

Just to be sure, I also extended the `num_samples` just to be on the safe side.